edge-util/src/Microsoft.Azure.Devices.Edge.Storage/IStorageSpaceChecker.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Storage
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public interface IStorageSpaceChecker
+    {
+        void SetMaxSize(long maxSize, Option<long> checkFrequency);
```

Comment:
nit: rename method since the method not only sets maxSize but also checkFrequency

Reply:
Not applicable anymore as we're not setting checkFrequency through this method.